crates/rlp/src/encode.rs

```diff
@@ -338,6 +338,39 @@ where
     }
 }
 
+// Encoding a list of items that are not the same static size
```

Comment:
I think this documentation is a bit misleading.

can we update this a bit that this accepts a slice of `dyn Encodable` and maybe add the test as an example to the docs.

@DaniPopes I'm not sure accepting a slice of boxed dyn is necessarily the best we can do here because this requires boxing the owned value.
wdyt

Reply:
thanks for the feedback, I assumed you'd want to alter this implementation just wanted to bring up the lack of encode_list that supports a dynamic vector.